packages/axelar-soroban-std/src/events.rs

```diff
@@ -1,13 +1,15 @@
 use core::fmt::Debug;
-use soroban_sdk::{Env, IntoVal, Topics, Val};
+use soroban_sdk::{Env, IntoVal, Topics, Val, Vec};
 
 #[cfg(any(test, feature = "testutils"))]
 pub use testutils::*;
 
 pub trait Event: Debug + PartialEq {
     fn topics(&self, env: &Env) -> impl Topics + Debug;
 
-    fn data(&self, env: &Env) -> impl IntoVal<Env, Val> + Debug;
+    fn data(&self, env: &Env) -> impl IntoVal<Env, Val> + Debug {
```

Comment:
```suggestion
    /// A default empty tuple/vector is used for event data, since majority of events only use topics.
    fn data(&self, env: &Env) -> impl IntoVal<Env, Val> + Debug {
```
explain the reasoning